open_flamingo/train/data.py

```diff
@@ -14,6 +14,7 @@
 import webdataset as wds
 from PIL import Image
 import base64
+from scipy import linear_sum_assignment
```

Comment:
I think this has to be from scipy.optimize import linear_sum_assignment? Correct me if I am wrong tho.

Reply:
oh sorry for the mistake, fixed it, thanks